java/src/jmri/jmrit/automat/AbstractAutomaton.java

```diff
@@ -106,22 +106,32 @@ public AbstractAutomaton(String name) {
     private volatile boolean threadIsStopped = false;
 
     /**
-     * Start this automat processing.
+     * Start this automat's processing loop.
      * <p>
-     * Overrides the superclass method to do local accounting.
+     * Will execute the init() method, if present, then
+     * repeatedly execute the handle() method until complete.
      */
     public void start() {
+        log.trace("start() invoked");
         if (currentThread != null) {
-            log.error("Start with currentThread not null!");
+            log.error("Start with a thread already running!");
         }
         currentThread = jmri.util.ThreadingUtil.newThread(this, name);
         currentThread.start();
         summary.register(this);
         count = 0;
+        log.trace("start() ends");
     }
 
     private volatile boolean running = false;
 
+    /**
+     * Is the thread in this object currently running?
+     *<p>
```

Comment:
Static analysis Java21 fails on this line: `warning: empty <p> tag`
It's not needed between the subject and the `@return`.